package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Seneca RabbitMQ transport",
   "main": "rabbitmq-transport.js",
   "scripts": {
-    "test": "node_modules/.bin/mocha test/*.test.js"
+    "test": "./node_modules/.bin/lab test/*test.js  -r console -v",
```

Comment:
We cheat and use ESLint with a 'Standard' setup. See https://github.com/senecajs/seneca-level-store